scripts/common/rook.sh

```diff
@@ -67,58 +67,76 @@ function remove_rook_ceph() {
     # make sure there aren't any PVs using rook before deleting it
     all_pv_drivers="$(kubectl get pv -o=jsonpath='{.items[*].spec.csi.driver}')"
     if echo "$all_pv_drivers" | grep "rook" &>/dev/null ; then
-        # do stuff
-        printf "%b" "$RED"
-        printf "ERROR: \n"
-        printf "There are still PVs using rook-ceph.\n"
-        printf "Remove these PVs before continuing.\n"
-        printf "%b" "$NC"
-        exit 1
+        logFail "There are still PVs using rook-ceph."
+        logFail "Remove these PV(s) before continuing:"
+        logFail "$all_pv_drivers"
+        return 1
     fi
 
     # scale ekco to 0 replicas if it exists
     if kubernetes_resource_exists kurl deployment ekc-operator; then
         kubectl -n kurl scale deploy ekc-operator --replicas=0
         log "Waiting for ekco pods to be removed"
-        if ! spinner_until 120 ekco_pods_gone; then
-             logFail "Unable to scale down ekco operator"
-             return 1
+        if ! spinner_until 300 ekco_pods_gone; then
+             logWarn "Unable to scale down ekco operator"
+             logWarn "The script will continue to try remove Rook"
```

Comment:
Are you sure this will still work?

Reply:
To me it's pretty likely that if you can't scale down ekco there's probably issues with the cluster

Reply:
It is unlike we be unable to scale down ekco. However, in this case why we should stop and does not try to delete?
Could not the next steps work?